tests/test_dispatch_cli.py

```diff
@@ -172,9 +172,9 @@ async def test_list_command(  # pylint: disable=too-many-arguments
                 "create",
                 "829",
                 "test",
+                "BATTERY",
                 "in 1 hour",
                 "1h",
-                "BATTERY",
```

Comment:
Is this change due to the positional arguments change the order as now there might not be a duration? I just want to be sure I got this correctly

Reply:
Yep, this change is due to the fact that it can now be skipped, so I moved it to the end (as those parameters are position based, not named based)